tasks/managed/upload-sbom-to-atlas/upload-sbom-to-atlas.yaml

```diff
@@ -325,21 +325,21 @@ spec:
           sbom_id="$(basename -s .json "$sbom_path")"
           supported_version_of_sbom="${sbom_path}.supported_version"
 
-          bombasticApiUrl=$(params.bombasticApiUrl)
-          echo "Uploading SBOM to $bombasticApiUrl (with id=$sbom_id)"
+          atlasApiUrl=$(params.atlasApiUrl)
+          echo "Uploading SBOM to $atlasApiUrl ($sbom_id)"
 
           handle_atlas_failure () {
             >&2 echo "WARNING: SBOM upload of $1 to Atlas has failed! Upload will be retried later."
             cp "$1" "${failedSbomDir}"
           }
 
-          curl -X PUT "${curl_opts[@]}" \
+          curl -X POST "${curl_opts[@]}" \
             --retry-max-time "$retry_max_time" \
             -H "authorization: Bearer $access_token" \
             -H "transfer-encoding: chunked" \
             -H "content-type: application/json" \
             --data "@$supported_version_of_sbom" \
-            "$bombasticApiUrl/api/v1/sbom?id=$sbom_id" \
+            "$atlasApiUrl/api/v2/sbom" \
```

Comment:
The test-upload-sbom-to-atlas-curl-fail test fails because of this - see mocks.sh - it simulates a failing atlas api call based on the api url string - you'll have to modify it there.

Reply:
This should be fixed now.